rewrite-java-core/src/test/java/ResponseTest.java

```diff
@@ -0,0 +1,51 @@
+import org.intellij.lang.annotations.Language;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.ChangeType;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+/**
+ * ResponseTest is used to test out the recipe that changes
+ * references to com.azure.core.http.rest.response to
+ * io.clientcore.core.http.models.response.
+ * @author Ali Soltanian Fard Jahromi
+ */
+public class ResponseTest implements RewriteTest {
+
+    /**
+     * This method sets which recipe should be used for testing
+     * @param spec stores settings for testing environment; e.g. which recipes to use for testing
+     */
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipeFromResource("/META-INF/rewrite/rewrite.yml",
+                "com.azure.rewrite.java.core.MigrateAzureCoreSamplesToAzureCoreV2");
+    }
+
+    /**
+     * This test method is used to make sure that the Response type is updated to the new version
+     */
+    @Test
+    void testUpdateResponseImport() {
+        @Language("java") String before = "import com.azure.core.http.rest.Response;\n" +
+                "import com.azure.core.util.BinaryData;";
+        before += "\npublic class Testing {";
+        before += "\n  public Testing(){";
+        before += "\n    Response<BinaryData> bdr = null;";
```

Comment:
do we need the body if only testing for import replace?

Reply:
Added type in the code that is being used for the unit test so that it tests the import replace and type replace.